fragments/master.sh

```diff
@@ -209,50 +226,9 @@ start_k8s(){
             --cluster-dns=10.0.0.10 \
             --cluster-domain=cluster.local \
             --allow-privileged=true --v=2 \
-            --pod-infra-container-image=${PAUSE_IMAGE}
-}
-
-run_addons_container(){
-  docker run \
```

Comment:
ok, this need additional cleanup then. We also have to think how to enhance the existing add-on manager.